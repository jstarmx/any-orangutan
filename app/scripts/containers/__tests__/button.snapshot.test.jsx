import React from 'react';
import renderer from 'react-test-renderer';

import { Button } from '../button';

it('renders correctly', () => {
  const tree = renderer.create(
    <Button activeFilter="all" label="favourite" filter={ () => {} } />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
