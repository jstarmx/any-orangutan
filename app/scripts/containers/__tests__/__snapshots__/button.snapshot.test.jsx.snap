// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="filter__button"
  onClick={[Function]}
  type="button"
>
  favourite
</button>
`;
